src/core/live_trackers/cpp/live_message_channel_tracker_impl.cpp

```
@@ -0,0 +1,238 @@
+#include <cstring>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <utility>
+
+namespace core
+{
+
+LiveMessageChannelTrackerImpl::LiveMessageChannelTrackerImpl(const OpenXRSessionHandles& handles,
+                                                             const MessageChannelTracker* tracker)
+    : handles_(handles), tracker_(tracker), receive_buffer_(tracker->max_message_size(), 0)
```

Cppcheck (2.20.0)

[style] The function 'required_extensions' is never used.

(unusedFunction)